pages/features.html

```diff
@@ -321,26 +387,80 @@ <h2>
 							item.style.zIndex = '3';
 							item.style.transform = 'translateX(100%)';
 						}
-						else  {
+						else {
```

Comment:
Let's fix the issue (of else being on a new line) instead of reproducing it.

```suggestion
						} else {
```